src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int number = random.nextInt(100);

```

Comment:
100 should be a constant field

also, we can declare new ColorSupplier() and new Random() as fields. so that we don't create a new object each time the method is called